src/Microsoft.ML.Sweeper/Parameters.cs

```diff
@@ -11,12 +11,11 @@
 using Microsoft.ML.CommandLine;
 using Microsoft.ML.Internal.Utilities;
 using Microsoft.ML.Sweeper;
-using Float = System.Single;
 
 [assembly: LoadableClass(typeof(LongValueGenerator), typeof(LongParamOptions), typeof(SignatureSweeperParameter),
     "Long parameter", "lp")]
 [assembly: LoadableClass(typeof(FloatValueGenerator), typeof(FloatParamOptions), typeof(SignatureSweeperParameter),
-    "Float parameter", "fp")]
+    "float parameter", "fp")]
```

Comment:
>float [](start = 5, length = 5)

I would probably desist from changing this particular one.

Reply:
Good catch! I thought it was in reference to the `Float` type parameter that we were getting rid of.